src/controllers/users.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const { UserService } = require('../services/users.service');
+
+const getAllUsers = (req, res) => {
+  const userService = new UserService();
+  const users = userService.findAll();
+
+  res.send(users);
+};
+
+const getOneUser = (req, res) => {
+  const userService = new UserService();
+
+  const { userId } = req.params;
+  const user = userService.findById(userId);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
```

Comment:
Did you check the swagger for responses?

You need to send status 400, when you don't have userId in this request.
You need to send status 404, when you don't have a user with provided userId.
And you need to send status 200 with the user in the payload when everything is okay.

![image](https://github.com/mate-academy/node_accounting-app-v2/assets/84296041/67f811c6-dea5-4f30-bdec-f0d668c4160f)

```
return 404 with any message if expected entity doesn't exist.
return 400 with any message if required parameter is not passed.
```

Let's add some messages to the errors.